src/main/java/bio/terra/service/snapshot/flight/create/CreateSnapshotFireStoreDataStep.java

```diff
@@ -69,9 +69,8 @@ public StepResult doStep(FlightContext context) throws InterruptedException {
             int numFilesSeen = 0;
             for (SnapshotMapTable mapTable : snapshotSource.getSnapshotMapTables()) {
                 for (SnapshotMapColumn mapColumn : mapTable.getSnapshotMapColumns()) {
-                    String fromDatatype = mapColumn.getFromColumn().getType();
-                    if (StringUtils.equalsIgnoreCase(fromDatatype, "FILEREF") ||
-                        StringUtils.equalsIgnoreCase(fromDatatype, "DIRREF")) {
+                    TableDataType fromDatatype = mapColumn.getFromColumn().getType();
+                    if (fromDatatype == TableDataType.FILEREF || fromDatatype == TableDataType.DIRREF) {
```

Comment:
Not necessarily for this PR but throwing an idea out there: Should there maybe be a helper method to detect it a type is in a class of types (e.g. `TableDataType.isFileSystemType(fromDataType)`). I could see that being useful for dates and numbers too)

Reply:
I guess it would need to be a utility class

Reply:
It would be nice but I don't think there's a way to add a method to a generated enum class. As a compromise maybe it could be a `Column` method, possibly most cases where we want to do this we're working with a datatype that's on a `Column`.

Reply:
Yeah, I didn't see that this was an autogenerated class when I first made the comment.  I think that ultimately, we may want a utility class for type helpers that does things like what I mentioned here and maybe does the type conversion between this and bigquery/parquet/etc...

Reply:
I checked and there are three places where code is looking at the field `Column.type` and comparing it to the enum values is when a `Column` is used. Two of them are looking for either `FILEREF` or `DIRREF` and one is only looking for `FILEREF`. Both of these cases could be converted to methods on `Column` so the caller doesn't need to access the type directly, e.g., `Column.isReference()` and `Column.isFileReference()`. Not necessary for this PR but something to think about.